plugins/scaffolder-react/api-report.md

```diff
@@ -459,9 +459,9 @@ export type ScaffolderTaskStatus =
 // @public
 export interface ScaffolderUseTemplateSecrets {
   // (undocumented)
-  secrets: Record<string, string>;
+  secrets: JsonObject;
```

Comment:
I think that we're going to have to make this a breaking change as I don't see another way round it.

But only for this `scaffolder-react` package, and you can create a separate changeset for it I think with some instructions on what the breaking change is and ways to fix the breakage.

Reply:
Hey, don't I already have that changeset? A bit confused

Reply:
This will need updating to `major` https://github.com/backstage/backstage/pull/26143/files#diff-98ec8365b5c60e8473b1ed3f2d52ab59e1647e56bc8553114a470a4e7b4dbd46R2

Reply:
But the other changeset can stay as `minor`

Reply:
Done